### DISCUSSION TITLE
Compatible spring versions

### DISCUSSION THREAD

**Discussion by pashap786** [spring-projects/spring-retry#389] [Q&A]
What versions of spring does this work with?

**Comment by snicoll** [comment]
what is "this"? If you're asking what version of Spring you can use with a version of Spring Retry, Spring retry is part of the Spring Boot BOM and each entry is [documented](https://docs.spring.io/spring-boot/docs/current/reference/html/dependency-versions.html#appendix.dependency-versions.coordinates) per version.

**Reply by garyrussell** [reply]
Specifically, when not used with Spring Boot, 1.3.4 and later works with all supported Spring Framework versions before 6.0; 2.0.4 and later require Spring Framework 6.0.x and later (and Java 17).

https://spring.io/projects/spring-framework#support
